test/run-tests.py

```diff
@@ -752,18 +750,16 @@ def main(args):
   except KeyboardInterrupt:
     print('\nInterrupted testing\n')
 
-  status.Clear()
+  status.Done()
 
   ret = 0
-
   if status.failed:
     sys.stderr.write('**** FAILED %s\n' % ('*' * (80 - 14)))
     for info in status.failed_tests:
       sys.stderr.write('- %s\n    %s\n' % (info.GetName(),
                                            ' '.join(info.last_cmd)))
     ret = 1
 
-  status.Print()
```

Comment:
Ah, I remember why I added this. After this PR, the output is inconsistent when you run it:
```
$ test/run-tests.py parse
[+503|-0|%100] (1.43s) test/parse/string-hex.txt (roundtrip fold-exprs)
$ test/run-tests.py parse
[+529|-0|%100] (1.37s) test/parse/string-hex.txt (roundtrip fold-exprs)
```